lopper.py

```diff
@@ -993,6 +993,14 @@ def exec_lop( self, lops_fdt, lop_node_number, options = None ):
                                     if not node_regex:
                                         if sl in selected_nodes:
                                             selected_nodes.remove( sl )
+                                    else: # here add prop
+                                        for selected_node in selected_nodes_possible:
+                                          selected_node + test_prop
```

Comment:
Select shouldn't modify the nodes. In my next set of updates, I'll start warning on the older style modify / add / delete operations if they use a regex and modify. The lops are moving to be more atomic, and hence it should be: select -> do something -> reset -> select -> do_something -> compound select -> do something else. 

Adding a property to the selected node is a side effect that we don't want. Select is just about tagging a series of nodes.

Reply:
so instead, should this be done inside of a plugin? or is there a suggested flow for adding properties to an existing node outside of the plugin?